spring-integration-core/src/main/resources/org/springframework/integration/config/spring-integration-4.3.xsd

```diff
@@ -2691,6 +2691,34 @@
 		</xsd:complexType>
 	</xsd:element>
 
+	<xsd:element name="stream-transformer">
+		<xsd:annotation>
+			<xsd:documentation>
+				Defines a Consumer Endpoint for the
+				'org.springframework.integration.transformer.StreamTransformer'
+				that converts an 'InputStream' payload to a byte[] or String.
+				Providing a 'charset' signals that the conversion to String is
+				required.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:extension base="specialized-transformer-type">
```

Comment:
Oh! Sorry, Gary. I didn't mean that you had made a mistake.
I just proposed to review (and maybe fix) that `specialized-transformer-type`, because it allows (by mistake?) any nested tags for awhile, not only `<poller>`.
That is my concern and I need here one more pair of eyes to be sure that I'm not insane :smile: